src/modules/user/controller/index.js

```diff
@@ -127,6 +127,7 @@ async function loginUser(req, res) {
 
   const token = jwt.sign({ userId: user.id }, process.env.JWT_SECRET_KEY);
   const userEmail = user.email;
+  sendEmails(userEmail);
```

Comment:
So as not to repeat, you can directly send user.email in the function (sendEmails) params.